eventdata/parameter_sources/randomevent.py

```diff
@@ -276,29 +286,58 @@ def generate_event(self):
         self._request.add_fields(event)
 
         # set host name
-        event["hostname"] = "web-{}-{}.elastic.co".format(event["geoip_continent_code"],random.randrange(1,3))
-
-        line = '{"@timestamp": "%s", ' \
-               '"offset":%s, ' \
-               '"source":"/usr/local/var/log/nginx/access.log","fileset":{"module":"nginx","name":"access"},"input":{"type":"log"},' \
-               '"beat":{"version":"6.3.0","hostname":"%s","name":"%s"},' \
-               '"prospector":{"type":"log"},' \
-               '"nginx":{"access":{"user_name": "-",' \
-               '"agent":"%s","user_agent": {"major": "%s","os": "%s","os_major": "%s","name": "%s","os_name": "%s","device": "%s"},' \
-               '"remote_ip": "%s","remote_ip_list":["%s"],' \
-               '"geoip":{"continent_name": "%s","city_name": "%s","country_name": "%s","country_iso_code": "%s","location":{"lat": %s,"lon": %s} },' \
-               '"referrer":"%s",' \
-               '"url": "%s","body_sent":{"bytes": %s},"method":"%s","response_code":%s,"http_version":"%s"} } }' % \
-               (event["@timestamp"],
-                event["offset"],
-                event["hostname"],event["hostname"],
-                event["agent"], event["useragent_major"], event["useragent_os"], event["useragent_os_major"], event["useragent_name"], event["useragent_os_name"], event["useragent_device"],
-                event["clientip"], event["clientip"],
-                event["geoip_continent_name"], event["geoip_city_name"], event["geoip_country_name"], event["geoip_country_iso_code"], event["geoip_location_lat"], event["geoip_location_lon"],
-                event["referrer"],
-                event["request"], event["bytes"], event["verb"], event["response"], event["httpversion"])
-
-        return line, index_name, self._type
+        event["hostname"] = "web-{}-{}.elastic.co".format("global",random.randrange(1,3))
+
+        line = {
```

Comment:
Rethink about this, see https://github.com/elastic/rally-eventdata-track/pull/5 for the reason it was set as string